index.html

```diff
@@ -559,19 +571,10 @@ <h2><dfn>disconnect()</dfn> method</h2>
         "WHATWG-DOM#context-object">context object</a>'s <a>observer
         buffer</a>.</p>
       </section>
-      <section>
+      <section data-link-for="PerformanceObserver">
         <h2><a>supportedEntryTypes</a> attribute</h2>
-        <p>The user agent MUST maintain <dfn>supported entry types</dfn>, a list
-        of strings representing the entry types which the user agent supports
-        for the <a>PerformanceObserver</a> interface. This list is populated by
-        specifications that define new entry types via the
-        <a>register a performance entry type</a> algorithm.</p>
-        <p>When <dfn>supportedEntryTypes</dfn>'s attribute getter is called, run the following steps:</p>
-        <ol>
-          <li>Let <var>result</var> be a copy of the user agent's <a>supported entry types</a>.</li>
-          <li>Sort <var>result</var> in alphabetical order.</li>
-          <li>Return <var>result</var>.</li>
-        </ol>
+        <p>When <dfn>supportedEntryTypes</dfn>'s attribute getter is called, it
+          MUST return the <a>relevant supported entry types</a>.</p>
```

Comment:
This doesn't work, unfortunately, because "relevant supported entry types" as defined above is only defined with reference to a platform object, but there is no platform object present here at all: it's a static attribute getter.

Reply:
Done